pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,183 @@
+# Running Buildkite Agent on Google Cloud Platform
+
+The Buildkite Agent can be run on [Google Cloud Platform](https://cloud.google.com). You may like to run the agent using individual VM instances on Google Compute Engine for fine control over long–lived agents. Or you may like to run a scalable cluster of agents using Docker and Kubernetes on the Google Container Engine to perform Docker–based builds.
+
+<%= toc %>
+
+## Running the agent on Google Compute Engine
+
+To run the agent on your own [Google Compute Engine](https://cloud.google.com/compute) instance use whichever installer matches your instance type. For example, launch an instance using the [latest Ubuntu LTS image](https://console.cloud.google.com/launcher?cat=OS&q=ubuntu) and follow [our Ubuntu setup instructions](/docs/agent/ubuntu).
+
+## Running the agent on Google Container Engine
+
+[Google Container Engine](https://cloud.google.com/container-engine) can also run the agent as a [Docker](https://www.docker.com) container using [Kubernetes](https://kubernetes.io). Make sure to run the container in [privileged mode](https://kubernetes.io/docs/user-guide/pods/#privileged-mode-for-pod-containers) and map the Docker socket and binary into the container as volumes if you wish to [run Docker–based builds](http://buildkite.dev/docs/builds/docker-containerized-builds). For example:
```

Comment:
The second sentence could start with something like "Before setting up the agent, ensure that your container is running in privileged mode". Then you could follow on with "If you are doing Docker-based builds, you'll also need to map the Docker socket and binary to the container as volumes".